cloudprober/values.yaml

```diff
@@ -20,6 +20,9 @@ image:
 configMap:
   name: ''
   create: true
+  # The config file suffix; supported suffixes are in
+  # https://github.com/cloudprober/cloudprober/blob/master/config/config.go#L72
```

Comment:
So the content of https://github.com/cloudprober/cloudprober/blob/master/config/config.go#L72 will change as config.go changes. Since we don't have a good authoritative source right now, I'd say mentioning the prefixes - .json (for file in JSON format), .yaml or .yml for YAML - will be better for now.

(I'll figure out a better way to do this -- we should not just rely on file extension)